data/notices.json

```diff
@@ -1,5 +1,21 @@
 {
   "notices": [
+    {
+      "title": "aws-wafv2: Reverted the naming pattern change where some structs changed from Foo to FooAction",
```

Comment:
I find it a bit confusing that we are focusing on what was reverted rather than what they will see.  In other words, `FooAction -> Foo` is easier to understand than `revert Foo -> FooAction`, even though logically they are equivalent.

Reply:
I see, ok will make the change.